### DISCUSSION TITLE
v0.39.0-20220416.182927

### DISCUSSION THREAD

**Discussion by qvacua** [qvacua/vimr#940] [General]
* Neovim 0.7.0 😀
* GH-922: Add an option to set font smoothing.
* GH-928: Bugfix: Files containing some special characters in their names cannot be opened.

<hr /><em>This discussion was created from the release <a href='https://github.com/qvacua/vimr/releases/tag/v0.39.0-20220416.182927'>v0.39.0-20220416.182927</a>.</em>

**Comment by kuntau** [comment]
Been waiting for this since the 0.7 release. Thank you 🙇🏽‍♂️
